### ISSUE TITLE
Couldn't find activation function mish, going with ReLU

### ISSUE THREAD

**Issue by francismontalbo** [AlexeyAB/darknet#6071]
Couldn't find activation function mish, going with ReLU

I've been getting this error whenever I try to run the new YoloV4

**Comment by yassinesefrioui** [comment]
i have the some error

**Comment by francismontalbo** [comment]
I works with yolov4-tiny I managed to train my own dataset. However, I'm having problem with the 'mish' activation.

**Comment by imaami** [comment]
I have a patch for this in a branch but I haven't made a pull request, I'll try to do that today.

**Comment by francismontalbo** [comment]
Hoping for an update on this soon. Thank you, guys!

**Comment by imaami** [comment]
A quick heads-up: I didn't have enough oomph left last night after a hot day and household chores to make the pull request happen. I'll do it in a couple hours as soon as I get another cup of coffee going and the air conditioning set up...

**Comment by francismontalbo** [comment]
Hello, was this fixed? Is mish now working? Thank you!

**Comment by francismontalbo** [comment]
Still getting the same error 

"Couldn't find activation function mish, going with ReLU"

**Comment by imaami** [comment]
> Still getting the same error
> 
> "Couldn't find activation function mish, going with ReLU"

No, sorry, I haven't had time for it yet. I'll let you know ASAP.

**Comment by francismontalbo** [comment]
Thank you my friend. Be safe.

**Comment by imaami** [comment]
@francismontalbo You know what, I don't know why you're seeing that error. I thought one of my bugfix branches deals with what you described, but I misremembered. The current unmodified master branch should be able to parse "activation=mish" in cfg files just fine.

I ran YOLOv4 inference on my computer, both with GPU enabled and without it, and I don't see the error at all. And looking at the source code I don't see any reason why it shouldn't work, either. The function `get_activation()` in `src/activations.c` - the only place that prints the error message you're getting - has recognized the word "mish" since commit bf8ea4183 in November 2019.

So either you're accidentally running an old build, or there's a bug in the parsing code that I'm not hitting, or maybe you have some local changes in the cfg file.

Can you please triple-check everything and give details about where you got the binary, etc.

**Comment by imaami** [comment]
> Thank you my friend. Be safe.

You too, thank you for the kind words!

**Comment by francismontalbo** [comment]
Thank you for the quick response my good friend. This helps me a lot. Btw is there a way for me to download an updated build of the darknet.exe? I previously used the one I found before, which works perfectly with yolov4-tiny but not with yolov4 due to the mish function. Would be helpful if I could find an updated build just like before. Thank you!

**Comment by francismontalbo** [comment]
Sorry for bothering.

**Comment by imaami** [comment]
> Thank you for the quick response my good friend. This helps me a lot. Btw is there a way for me to download an updated build of the darknet.exe? I previously used the one I found before, which works perfectly with yolov4-tiny but not with yolov4 due to the mish function. Would be helpful if I could find an updated build just like before. Thank you!

I'm building a bugfix branch right now, I assume you're on Windows? I'll post a download link as soon as I can.

**Comment by imaami** [comment]
@francismontalbo Try this: https://github.com/imaami/darknet/suites/859871252/artifacts/9898103

**Comment by Maioy97** [comment]
I got the same error in the now latest version of darknet, I pulled it on April 28th 

any updates ?
